packages/example/src/layouts/AppHeader/AppHeader.tsx

```diff
@@ -1,5 +1,8 @@
 import React from "react";
 import clsx from "clsx";
+import AppBar from "@mui/material/AppBar";
+import Container from "@mui/material/Container";
+import Toolbar from "@mui/material/Toolbar";
```

Comment:
I think we can do like this here:

```ts
import { AppBar, Container, Toolbar } from '@mui/material';
```

Reply:
It is optimization for dev environment https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
interesting